src/Microsoft.Win32.Registry/src/project.json

```diff
@@ -1,27 +1,27 @@
 {
   "frameworks": {
-    "net46": {
+    "net463": {
       "dependencies": {
         "Microsoft.TargetingPack.NETFramework.v4.6": "1.0.1"
       }
     },
-    "netstandard1.3": {
+    "netstandard1.7": {
       "dependencies": {
         "Microsoft.NETCore.Platforms": "1.0.1",
-        "System.Collections": "4.0.10",
-        "System.Diagnostics.Debug": "4.0.10",
-        "System.Diagnostics.Tools": "4.0.0",
-        "System.Globalization": "4.0.10",
-        "System.IO": "4.0.10",
-        "System.Resources.ResourceManager": "4.0.0",
-        "System.Runtime": "4.0.20",
-        "System.Runtime.Extensions": "4.0.10",
-        "System.Runtime.Handles": "4.0.0",
-        "System.Runtime.InteropServices": "4.0.20",
-        "System.Threading.Tasks": "4.0.10"
+        "System.Collections": "4.3.0-beta-24604-02",
+        "System.Diagnostics.Debug": "4.3.0-beta-24604-02",
+        "System.Diagnostics.Tools": "4.3.0-beta-24604-02",
+        "System.Globalization": "4.3.0-beta-24604-02",
+        "System.IO": "4.3.0-beta-24604-02",
+        "System.Resources.ResourceManager": "4.3.0-beta-24604-02",
+        "System.Runtime": "4.3.0-beta-24604-02",
+        "System.Runtime.Extensions": "4.3.0-beta-24604-02",
+        "System.Runtime.Handles": "4.3.0-beta-24604-02",
+        "System.Runtime.InteropServices": "4.3.0-beta-24604-02",
+        "System.Threading.Tasks": "4.3.0-beta-24604-02"
       },
       "imports": [
```

Comment:
>imports [](start = 7, length = 7)

The imports section is not required -- they can be removed from all project.json files.